content/influxdb/cloud-serverless/write-data/api/v1-http.md

```diff
@@ -0,0 +1,294 @@
+- {{% code-placeholder-key %}}`RETENTION_POLICY`{{% /code-placeholder-key %}}: the [retention policy](#map-databases-and-retention-policies-to-buckets)
+- {{% code-placeholder-key %}}`API_TOKEN`{{% /code-placeholder-key %}}: a [token](/influxdb/cloud-serverless/admin/tokens/) with sufficient permissions to the mapped bucket
+
+##### Other Telegraf configuration options
+
+`influx_uint_support`: supported in InfluxDB IOx.
+
+For more plugin options, see [`influxdb`](https://github.com/influxdata/telegraf/blob/master/plugins/outputs/influxdb/README.md) on GitHub.
+
+#### Interactive clients
+
+To test InfluxDB v1 API writes interactively from the command line, use common HTTP clients such as cURL and Postman.
+
+{{% note %}}
+For production use cases, use tools such as [client libraries](#client-libraries) that construct line protocol for you and provide batch writing options.
+{{% /note %}}
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[Token Auth](#)
+[Basic Auth](#)
+[Query String Auth](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+{{% code-placeholders "DATABASE_NAME|API_TOKEN|RETENTION_POLICY" %}}
+```sh
+curl -i "https://{{< influxdb/host >}}/write?db=DATABASE_NAME&rp=RETENTION_POLICY&precision=s" \
+    --header "Authorization: Token API_TOKEN" \
+    --header "Content-type: text/plain; charset=utf-8" \
+    --data-binary 'home,room=kitchen temp=72 1463683075'
+```
+{{% /code-placeholders %}}
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+{{% code-placeholders "DATABASE_NAME|API_TOKEN|RETENTION_POLICY" %}}
+```sh
+curl -i "https://{{< influxdb/host >}}/write?db=DATABASE_NAME&rp=RETENTION_POLICY&precision=s" \
+    -user "ignored":"API_TOKEN" \
+    --header "Content-type: text/plain; charset=utf-8" \
+    --data-binary 'home,room=kitchen temp=72 1463683075'
+```
+{{% /code-placeholders %}}
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+{{% code-placeholders "DATABASE_NAME|API_TOKEN|RETENTION_POLICY" %}}
+```sh
+curl -i "https://{{< influxdb/host >}}/write?db=DATABASE_NAME&rp=RETENTION_POLICY&precision=s" \
+    --header "Content-type: text/plain; charset=utf-8" \
+    --data-urlencode "u=ignored" \
+    --data-urlencode "p=DATABASE_TOKEN" \
+    --data-binary 'home,room=kitchen temp=72 1463683075'
+```
+{{% /code-placeholders %}}
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+Replace the following:
+
+- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: the [database](#map-databases-and-retention-policies-to-buckets)
+- {{% code-placeholder-key %}}`RETENTION_POLICY`{{% /code-placeholder-key %}}: the [retention policy](#map-databases-and-retention-policies-to-buckets)
+- {{% code-placeholder-key %}}`API_TOKEN`{{% /code-placeholder-key %}}: a [token](/influxdb/cloud-serverless/admin/tokens/) with sufficient permissions to the mapped bucket
+
+#### v1 CLI (not supported)
+
+Don't use the v1 CLI with {{% product-name %}}.
+While it may coincidentally work, it isn't officially supported.
```

Comment:
```suggestion
#### v1 influx CLI (not supported)

Don't use the v1 `influx` CLI with {{% product-name %}}.
While it may coincidentally work, it isn't officially supported.
```